charts/prometheus-json-exporter/Chart.yaml

```
@@ -19,13 +19,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.0.2"
+appVersion: "1.0.3"
```

why is this changed if the used docker image is the same?

---

I guess 0.3.0 would be the correct value here?
```suggestion
appVersion: "0.3.0"
```

---

and image tag could be renmoved, so app version is used: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-json-exporter/values.yaml#L11

---

This seems like an improvement that should be picked up separately no?

---

@rouke-broersma why? IMHO it's all Bugfix scope so no need to do it in another pr.

@schmiddim Take your time and get well soon :)

---

I have changed the version number